specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -1612,6 +1677,22 @@
                   "description": "Counts of various node states on the compute.",
                   "readOnly": true,
                   "$ref": "#/definitions/NodeStateCounts"
+                },
+                "sshPortPublicAccess": {
```

Comment:
>sshPortPublicAccess [](start = 17, length = 19)

i see we are adding bunch of new properties .. These need to be introduced in new api version , unless we have strong reasons for not to.

How would we handle scenarios where user is sending the PUT request with these properties & then without these ?
Do we clear or retain these properties?

Reply:
It is safe to introduce a new api version.

---
In reply to: [299261562](https://github.com/Azure/azure-rest-api-specs/pull/6291#discussion_r299261562) [](ancestors = 299261562)